helpers/general.rb

```diff
@@ -223,7 +223,7 @@ def check_visible_location
     #
     # If location not previously retrieved, require it be visible or tied to the current project
     # when retrieving it.  This is called when creating resources in the web routes.
-    @location ||= if (id = typecast_params.uuid("location"))
+    @location ||= if (id = typecast_params.ubid_uuid("location") || typecast_params.uuid("location"))
```

Comment:
You're the expert, but I presume that instead of erroring these nil out (thus become falsy) if they don't match?

Reply:
Correct.  If you wanted to force a match, and raise if there wasn't a match, you could use `ubid_uuid!`.  In this case, we have existing error handling directly below this line, so it's better to use that.